### DISCUSSION TITLE
Ricoh 201 (2550, 5200)

### DISCUSSION THREAD

**Discussion by shevvl64** [alexpevzner/sane-airscan#274] [General]
MFPs scan one sheet at a time well.
When scanning through an autofeeder, one sheet is scanned and an error appears.
It feels like there is no information about the end of the sheet and the beginning of a new one...

**Comment by alexpevzner** [comment]
Hi @shevvl64,

I want to look to the protocol trace logs. To enable protocol trace, please in the `/etc/sane.d/airscan.conf` file uncomment the following line:
```
[debug]
trace = ~/airscan/trace
```
Logs will be collected at the `~/airscan/trace` directory which will be created automatically. Only `.log` files are needed for no, `.tar` files are not required.

**Reply by shevvl64** [reply]
I have collected the logs as you requested.
[simple-scan-zeroconf.log](https://github.com/alexpevzner/sane-airscan/files/10756216/simple-scan-zeroconf.log)
[simple-scan-ricoh-201.log](https://github.com/alexpevzner/sane-airscan/files/10756218/simple-scan-ricoh-201.log)
